src/Datadog.Trace/Util/DomainMetadata.cs

```diff
@@ -10,6 +10,9 @@ internal static class DomainMetadata
     {
         private const string UnknownName = "unknown";
         private static Process _currentProcess;
```

Comment:
Looks like `_currentProcess` isn't used anymore and can be removed.

Reply:
I was using that as an indicator that we've initialized the type. I'll just replace this with an actual `_initialized` bool. Thanks for spotting this!

Reply:
Resolved in cbf5a666e4df5b16ba2009dde35bbb535a8230d3